src/server/services/wms/qgswms.cpp

```diff
@@ -76,6 +76,12 @@ namespace QgsWms
                                      QStringLiteral( "Please add or check the value of the REQUEST parameter" ), 501 );
         }
 
+        if ( wmsRequest.wmsParameters().version().isEmpty() )
```

Comment:
Can we have an exception for GetCapabilities request ?

Reply:
Actually I updated the PR to check the version parameter only in case of `getmap` and OGC tests seem green too.